test/functional/spec-runner/index.js

```diff
@@ -233,6 +233,7 @@ function runTestSuiteTest(configuration, spec, context) {
   // test-specific client options
   clientOptions.autoReconnect = false;
   clientOptions.haInterval = 100;
+  clientOptions.minHeartbeatFrequencyMS = 100;
```

Comment:
What's this change for?

Reply:
A majority of the retryable reads tests force some sort of error on the server, forcing the server to retry the operation. When a retry occurs, if no suitable server is found during server selection, the driver will issue an immediate check on all known servers. The SDAM specification, however, mandates that the check should be "debounced" by [maxHeartbeatFrequencyMS](https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst#multi-threaded-or-asynchronous-server-selection) (fixed value of 500ms) meaning each of these tests would wait 500ms between each attempt to read. Reducing this to 100ms in our tests just makes the tests run faster.